conf/messages

```diff
@@ -53,8 +53,10 @@ checkingRecords.antivirus=Anti-virus scanning
 checkingRecords.checksum=Validating data integrity
 checkingRecords.ffid=Identifying file formats
 fileChecksResults.header=Record check results
-fileChecksResults.title=Record check results
-fileChecksResults.description=Here are your results
+fileChecksResults.title=Checks Completed!
+fileChecksResults.description=Your folder ''{0}'' containing ''{1} items'' has been successfully checked and uploaded. Please click ''Continue'' to proceed with your transfer.
```

Comment:
This might be tricky, but can "items" be singular when there is only 1 item?

Also do we need the quotes around `{1} items`?

Reply:
Ah I didn't think about that case! I'll have a look and see if it can be changed when it's a singular. The quotation marks were around the both the number and the word items in the wireframes, but I do think it looks a bit strange. I can ask Rooha about it when she's back?